src/Project/ProjectService.ts

```diff
@@ -0,0 +1,25 @@
+import { INTERNAL_SERVER_ERROR, NOT_FOUND } from "../Resources/constants/statusCodes";
+import { getXataClient } from "../xata";
+import { HttpException } from "../Resources/exceptions/HttpException";
+const xata = getXataClient();
+
+class ProjectService {
+
+
+    async getProjectDetails(projectId: string, userId: string) {
+
+        const record = await xata.db.Project
+            .select(["assetController", "assetType", "enabledPaymentMethods.*", "web3Environment",
+                "treasuryWallet", "enabledPaymentMethods.*", "owner.walletAddress", "xata.createdAt"])
+            .filter({ "id": projectId, "owner.id": userId })
```

Comment:
It will be more effective if we filter before we select.

Reply:
@king-Alex-d-great It does not really matter when using Xata DB, but will update it sha.